### PR TITLE
Timer text color and sound

### PR DESCRIPTION
**Pull request by coreylevine**
Changed the timer text color when there is 10 seconds left.
Added a sound countdown when there is 5 seconds left.

### CODE REVIEW

File: app/src/main/java/org/liberty/android/fantastischmemo/ui/QuizActivity.java [hunk 1]
```diff
@@ -153,14 +154,28 @@ public void onCreate(Bundle savedInstanceState) {
     public void startTimer() {
         this.countDownTimer = new CountDownTimer(timeLeftInMilliseconds, 1000) {
 
+            MediaPlayer mediaPlayer = MediaPlayer.create(getApplicationContext(), R.raw.countdown);
+
             @Override
             public void onTick(long millisUntilFinished) {
                 timeLeftInMilliseconds = millisUntilFinished;
+
+                if(!mediaPlayer.isPlaying()) {
```
Comment (svelupillai) [inline app/src/main/java/org/liberty/android/fantastischmemo/ui/QuizActivity.java]: needs space after if

File: app/src/main/java/org/liberty/android/fantastischmemo/ui/QuizActivity.java [hunk 2]
```diff
@@ -153,14 +154,28 @@ public void onCreate(Bundle savedInstanceState) {
...
+                    if (timeLeftInMilliseconds <= 8000) {
+                        mediaPlayer.start();
+                    }
+                }
+
+                if(timeLeftInMilliseconds <= 11000){
```
Comment (svelupillai) [inline app/src/main/java/org/liberty/android/fantastischmemo/ui/QuizActivity.java]: need space after if and after closing parenthesis 
if () {

File: app/src/main/java/org/liberty/android/fantastischmemo/ui/QuizActivity.java (hunk 1 above)
Comment (coreylevine) [reply app/src/main/java/org/liberty/android/fantastischmemo/ui/QuizActivity.java]: Fixed!

File: app/src/main/java/org/liberty/android/fantastischmemo/ui/QuizActivity.java (hunk 2 above)
Comment (coreylevine) [reply app/src/main/java/org/liberty/android/fantastischmemo/ui/QuizActivity.java]: Fixed!

**Comment by svelupillai** [comment]
Favourite Espresso Test passes:
image

**Comment by svelupillai** [comment]
Hint Espresso Test passes:
image

**Comment by svelupillai** [comment]
HintToggleCHeckBox Espresso Test passes:
image

**Comment by j-suguma** [comment]
CI build for timerTextColor:
image

tests:
image

**Comment by svelupillai** [comment]
ShuffleTest Espresso Test passes:
image

**Pull request merged**